bliss/models/galaxy_encoder.py

```diff
@@ -50,17 +50,18 @@ def center_ptiles(
 class GalaxyEncoder(pl.LightningModule):
     def __init__(
         self,
+        decoder,
         hidden: int = 256,
-        decoder_kwargs: dict = None,
         optimizer_params: dict = None,
     ):
         super().__init__()
-        self.save_hyperparameters()
+        # prevent saving `decoder`
+        self.save_hyperparameters("hidden", "optimizer_params")
```

Comment:
is there a way we can still save decoder input arguments? are those still being saved elsewhere?